nucliadb/nucliadb/standalone/api_router.py

```diff
@@ -94,3 +95,29 @@ def get_temp_access_token(request: Request):
     jwetoken.add_recipient(jwk.JWK(**orjson.loads(settings.jwk_key)))
     token = jwetoken.serialize(compact=True)
     return JSONResponse({"token": token})
+
+
+@standalone_api_router.get("/cluster/nodes")
+async def node_members(request: Request) -> JSONResponse:
+    return JSONResponse(
+        [
+            {
+                "id": node.id,
+                "listen_address": node.address,
+                "type": node.label,
+                "shard_count": node.shard_count,
+                "dummy": node.dummy,
+            }
+            for node in manager.get_index_nodes()
+        ]
+    )
+
+
+@standalone_api_router.get("/health/alive")
+async def alive(request: Request) -> JSONResponse:
+    return JSONResponse({"status": "ok"})
+
+
+@standalone_api_router.get("/health/ready")
+async def ready(request: Request) -> JSONResponse:
```

Comment:
WDYT about returning an error here if the nodes can't discover each other (ie: the cluster can't be formed)?